src/components/PageDefinitions.js

```diff
@@ -257,11 +256,40 @@ class PageDefinitions extends AbstractPageDefinitions {
     this.props.dispatch(uiNotificationNew({ type: 'info', message, timeout: 5000 }))
   }
 
-  onDrop(acceptedFiles, rejectedFiles) {
+  onDragOver = e => e.preventDefault()
```

Comment:
Can we factor all of this out into a different component? For exapmle, we'll want to DnD onto the harvest page as well.